iOSTest/View Controllers/MenuViewController.swift

```diff
@@ -23,13 +23,43 @@ class MenuViewController: UIViewController {
      */
     
     // MARK: - Outlets
-    
+
+    @IBOutlet weak var chatButton: UIButton!
+    @IBOutlet weak var loginButton: UIButton!
+    @IBOutlet weak var animationButton: UIButton!
     
     // MARK: - Lifecycle
     override func viewDidLoad() {
         
         super.viewDidLoad()
         
+        title = "Coding Tasks"
+        
+        // Round the corners of all buttons
+        chatButton.layer.cornerRadius = 10.0
+        loginButton.layer.cornerRadius = 10.0
+        animationButton.layer.cornerRadius = 10.0
+        
+        // Set the button insets
+        
+        let imageSpacing = CGFloat(22.0)
+        let titleSpacing = CGFloat(16.0)
+        let netTitleSpacing = imageSpacing + titleSpacing
+        
+        // Add padding to the chat button
+        chatButton.imageEdgeInsets = UIEdgeInsets(top:0.0, left: imageSpacing, bottom: 0.0, right: 0.0)
+        
+        chatButton.titleEdgeInsets = UIEdgeInsets(top:0.0, left: netTitleSpacing, bottom:0.0, right: 0.0)
+        
+        // Add padding to the login button
+        loginButton.imageEdgeInsets = UIEdgeInsets(top:0.0, left: imageSpacing, bottom: 0.0, right: 0.0)
+        
+        loginButton.titleEdgeInsets = UIEdgeInsets(top:0.0, left: netTitleSpacing, bottom:0.0, right: 0.0)
+        
+        // Add padding to the animation button
+        animationButton.imageEdgeInsets = UIEdgeInsets(top:0.0, left: imageSpacing, bottom: 0.0, right: 0.0)
+               
+        animationButton.titleEdgeInsets = UIEdgeInsets(top:0.0, left: netTitleSpacing, bottom:0.0, right: 0.0)
```

Comment:
Avoid from cluttering the `viewDidLoad` and place all of this functionality in a function. It'll look cleaner to have a single function name rather than setting properties in `viewDidLoad`. As a rule of thumb, set properties in `viewDidLoad` sparingly.